templates/web_gateway/schema.capnp

```diff
@@ -7,6 +7,7 @@ struct Schema {
     farmerIyoOrg @3 :Text; # farmer for nodes to create etcd instances on
     publicNode @4 :Text; # node to deploy traefik and coredns on
     publicIps @5 :List(Text); # list of public ips to be used with this web gateway
+    etcdConnectionInfo @6 :Text; #save last etcd connection info
```

Comment:
Here you specified the type of this property as a string, but in the code you actually store some object in it